tests/ccl_test_distances.py

```diff
@@ -111,139 +112,139 @@ def Neff_from_N_ur_N_ncdm(N_ur, N_ncdm):
 path = dirname(abspath(__file__))
 def read_chi_benchmark_file():
     """
-    Read the file containing all the radial comoving distance benchmarks 
+    Read the file containing all the radial comoving distance benchmarks
     (distances are in Mpc/h)
     """
     # Load data from file
     dat = np.genfromtxt(join(path,"benchmark/chi_model1-5.txt")).T
-    assert(dat.shape == (6,6))
-    
+    assert_(dat.shape == (6,6))
```

Comment:
I found these weird uses of assert so I went ahead and fixed them. Not related to this PR. Sorry. Bad form on my part.

Reply:
Not sure what was wrong with the original line here.  `assert` and `numpy.assert_` are practically synonyms.  I don't see any reason to prefer the latter.